### ISSUE TITLE
Clubs and Organization time order sort

### ISSUE THREAD

**Issue by C13L0** [phsc-edu/2018#122]
@jeppy64 Please feel free to document information on restructuring the events. I changed the title and moved to Phase II for further development

---------
@jeppy64 

https://student-life.phsc.edu/
Club and Organization sorts properly by date but not by time. Please sort by date then time.

Thanks!

![time-order](https://user-images.githubusercontent.com/3138019/38433192-f7f61c62-3997-11e8-91d5-2ac31d9dca85.png)

**Comment by C13L0** [comment]
Removing estimate until #255 is completed

**Comment by jeppy64** [comment]
@C13L0 
this ticket has been completed against ticket #255 

I originally didn't include migrating all club and organization nodes, but while in the configuration move I updated the organization schedule bundle with the new field formatters and updated all views as well. Add to calendar function was also added. To ensure data integrity and back end handling, all organization schedule nodes have been updated on production.

additionally, production sites were cloned down to dev so we have a before/after

time to complete: 6 hours
